pkg/distributor/distributor_test.go

```diff
@@ -4230,10 +4282,24 @@ type mockIngester struct {
 	circuitBreakerOpen            bool
 }
 
-func (i *mockIngester) address() string {
+func (i *mockIngester) instanceID() string {
 	return fmt.Sprintf("ingester-%s-%d", i.zone, i.id)
 }
 
+// partitionID returns the partition ID owned by this ingester when ingest storage is used.
+func (i *mockIngester) partitionID() int32 {
+	id, err := ingest.IngesterPartitionID(i.instanceID())
+	if err != nil {
+		panic(err)
+	}
+
+	return id
```

Comment:
isn't this just the same as 
```suggestion
	return i.id
```
?

Reply:
Yes, it definitely it. The reason I've used `IngesterPartitionID()` is (a) to make the intention clear and (b) to spot this testing place when we look for `IngesterPartitionID()` references.